models/Post.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PostSchema = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'User'
+  },
+  content: {
+    type: String,
+    required: true
+  },
+  title: {
+    type: String
+  },
+  date: {
+    type: Date,
+    default: Date.now()
+  },
+  comments: [
```

Comment:
Did you consider making comments separate documents with their own schema? There are pros and cons to this, and I think ultimately the way you're doing it is probably best, but I would be curious to hear what made you pick it :)